apps/aecore/src/aec_chain_state.erl

```diff
@@ -556,7 +544,7 @@ calculate_state_trees(Node, Acc, State) ->
         {ok, Trees} when Acc =:= [] -> {stored, Trees};
         {ok, Trees} -> {calculated, apply_node_transactions(Acc, Trees, State)};
         error when IsGenesis  ->
-            Trees = genesis_state_tree(Node),
+            Trees = aec_blocks:trees(Node#node.content),
```

Comment:
I see this removed hardcoded expectation of genesis having no txs. I need to understand this better.

Reply:
The hardcoded expectation on no txs in genesis justified the lack of call to [`apply_node_transactions`](https://github.com/velzevur/epoch/blob/19e60155719581dda3196912ae4490452381bc11/apps/aecore/src/aec_chain_state.erl#L564) as [called in the case 2 lines above](https://github.com/aeternity/epoch/pull/399/files#diff-4ac652308bacbca71e5f388759d362c1R545). Please re-instate hardcode expectation on no txs.

Reply:
I second adding back the assertion of an empty transactions list. It works for now, but it is better to be safe for the future.